hardBanking/main.swift

```diff
@@ -196,23 +218,28 @@ func summary() -> Int? {
 }
 
 //14. Длинное словоы
-//func moreHigh(_ string: String) -> String {
-//    let word = string.split(separator: " ")
-//    guard word.count > 1 else { return string }
-//    let maxElement = word.count
-//    var itIsHigh = String()
-//    for indexOne in 0..<maxElement - 1 {
-//        for indexTwo in indexOne+1..<maxElement {
-//            if word[indexOne].count > word[indexTwo].count {
-//                itIsHigh = String(word[indexOne])
-//            } else if word[indexOne].count < word[indexTwo].count {
-//                itIsHigh = String(word[indexTwo])
-//            }
-//        }
-//    }
-//    return itIsHigh
-//}
-// Эта функция не работает так, как надо
+func moreHigh(_ string: String) -> String {
+let word = string.split(separator: " ")
+    guard word.count > 1 else { return string }
+    let maxElement = word.count
+    var itIsHigh = String()
+    var newArray = [String]()
+    for i in word {
+        newArray.append(String(i))
+        print(newArray)
+    }
+    for (indexOne,valueOne) in newArray.enumerated(){
+        print("\(indexOne) -> \(valueOne)")
+        for (indexTwo,valueTwo) in newArray.enumerated(){
+            if indexOne != indexTwo {
+                if valueOne.count > valueTwo.count {
+                    newArray.remove(at: indexTwo)
+                }
+            }
+        }
+    }
+    return newArray[0]
+}
```

Comment:
> Можно быо вот так сделать   

    func longestString(string: String) -> String {
        let words = string.split(separator: " ")
        var longestWord = ""
    
    for word in words {
        if word.count > longestWord.count {
            longestWord = String(word)
        }
    }
    
    return longestWord
    }